test/khash_test.erl

```diff
@@ -0,0 +1,443 @@
+-module(khash_test).
+
+-compile([export_all]).
+
+-define(NODEBUG, true).
```

Comment:
Ok, I found how I miss this moment early - it's because Erlang 18. Same rebar 2.5.1:

```
{ok,{khash_test,[{compile_info,[{options,[{outdir,".eunit"},
                                          {d,'TEST'},
                                          debug_info,
                                          {i,"include"}]},
                                {version,"6.0"},
                                {time,{2015,7,15,6,8,10}},
                                {source,"/home/kxepal/projects/asf/couchdb/couchdb-khash/test/khash_test.erl"}]}]}}
```

With Erlang R14B01:
```
{ok,{khash_test,[{compile_info,[{options,[export_all,
                                          {outdir,".eunit"},
                                          {d,'TEST'},
                                          debug_info,
                                          {i,"include"},
                                          return_errors,return_warnings,export_all]},
                                {version,"4.7.2"},
                                {time,{2014,10,14,21,1,55}},
                                {source,"/root/couchdb/src/khash/test/khash_test.erl"}]}]}}
```

For the last one tests runs for the same 8 seconds, no warnings. For the Erlang 18 some major amount of tests seems to be skipped. The only major change for eunit was [hrl split](https://github.com/erlang/otp/commit/c574bd33c39d91c487c3fcd819226ecfc46c13c8) but I didn't notice how it could be related.